### PR TITLE
Bump master to 1.21.0-unstable

### PR DESCRIPTION
**Pull request by kamilchodola**
_Bump to 1.21.0-unstable_

#### What types of changes does your code introduce?

- [ ] Bugfix (a non-breaking change that fixes an issue)
- [ ] New feature (a non-breaking change that adds functionality)
- [ ] Breaking change (a change that causes existing functionality not to work as expected)
- [ ] Optimization
- [ ] Refactoring
- [ ] Documentation update
- [X] Build-related changes
- [ ] Other: _Description_

#### Requires testing

- [ ] Yes
- [X] No

#### If yes, did you write tests?

- [ ] Yes
- [X] No

### CODE REVIEW

File: src/Nethermind/Directory.Build.props [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <Copyright>Demerzel Solutions Limited</Copyright>
     <Product>Nethermind</Product>
     <SourceRevisionId Condition="'$(Commit)' != ''">$(Commit.Substring(0, 8))</SourceRevisionId>
-    <VersionPrefix>1.20.0-unstable</VersionPrefix>
+    <VersionPrefix>1.21.0-unstable</VersionPrefix>
```
Comment (rubo) [inline src/Nethermind/Directory.Build.props:12]: Should be:

```csproj
<VersionPrefix>1.21.0</VersionPrefix>
<VersionSuffix>unstable</VersionSuffix>
```

**Review by rubo — CHANGES_REQUESTED** [review]

File: src/Nethermind/Directory.Build.props (hunk 1 above)
Comment (kamilchodola) [reply src/Nethermind/Directory.Build.props:12]: @rubo Is this some recent change? we have this approach for some time already.

Comment (kamilchodola) [reply src/Nethermind/Directory.Build.props:12]: Where commit will be? Still it be like 1.21.0-unstable+commit?

Comment (rubo) [reply src/Nethermind/Directory.Build.props:12]: No changes in version format. This is just the way to do it. I've always missed this PRs to review. This time I caught it ;)

**Review by rubo — APPROVED** [review]

**Pull request merged**